types/index.d.ts

```diff
@@ -0,0 +1,36 @@
+import {Node} from 'unist'
+
+declare namespace sanitize {
+  /**
+   * Possible HTML attribute values
+   */
+  type HTMLValue = string | number | boolean
+
+  /**
+   * Sanitization configuration
+   */
+  interface Schema {
+    attributes?: {
```

Comment:
Adding doc comments to settings can also be helpful for adopters (especially in an IDE like VSCode).
For example, pulling the description from the _readme_:

```suggestion
    /**
     * Map of tag names to allowed property names
     */
    attributes?: {
```

the same goes for the other options/properties.

Reply:
@TaylorBeeston Is this something you’d want to add? If so, I’ll wait on merging for you to add them.

Reply:
Sure! I'll just need a couple days. Going on a trip for my wife's birthday this weekend